torch/ao/quantization/utils.py

```diff
@@ -223,35 +221,34 @@ def op_is_int8_dynamically_quantized(qconfig) -> bool:
     """ Given a qconfig, returns True if this op is using int8 dynamic
     quantization
     """
-    activation_dtype, weight_dtype, activation_compute_dtype = \
+    activation_dtype, weight_dtype = \
         get_qconfig_dtypes(qconfig)
     return (
-        activation_dtype is torch.float and
+        activation_is_int8_quantized(qconfig) and
```

Comment:
this is a bit confusing.  `activation` here is talking about the activation after quantization.  It's not talking about the reference model - that's an intermediate step which is not exposed to most users.